cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -2901,27 +2901,20 @@
                 "  proxy_args=\"\"\n",
                 "  touch /tmp/proxy.sh\n",
                 "fi\n",
-                "if [ \"${custom_cookbook}\" != \"NONE\" ]; then\n",
-                "  cookbook_url=${custom_cookbook}\n",
-                "else\n",
-                "  if [ \"",
-                {
-                  "Ref": "AWS::Region"
-                },
-                "\" == \"us-east-1\" ]; then\n",
-                "    s3_prefix=s3\n",
-                "  else\n",
-                "    s3_prefix=s3-",
+                "  _region=",
                 {
                   "Ref": "AWS::Region"
                 },
                 "\n",
+                "if [ \"${custom_cookbook}\" != \"NONE\" ]; then\n",
+                "  cookbook_url=${custom_cookbook}\n",
+                "else\n",
+                "  if [ \"${_region}\" =~ \"^cn-\" ]; then\n",
```

Comment:
Mind double `[[ ]]` and no quotes around the regex `if [[ \"${_region}\" =~ ^cn- ]];`